lib/sync/installation.js

```diff
@@ -102,14 +102,24 @@ module.exports.processInstallation = (app, queues) => {
       return processor(github, repository, cursor, perPage)
     }
 
+    const handleGitHubError = (err) => {
+      const ignoredErrors = [
+        'MAX_NODE_LIMIT_EXCEEDED',
+        'NOT_FOUND'
+      ]
+      const isIgnoredError = ignoredErrors.find(ignoredError => String(err).includes(ignoredError))
+
+      if (isIgnoredError) {
```

Comment:
Excuse me if I'm reading this wrong, but aren't we inverting the logic?

Before it was `!err.include?(MAX_EXCEEDED)` which says `isNotIgnoredError` `err.include?(MAX_EXCEEDED) || err.include?(NOT_FOUND)` which is `isIgnoredError`.

It's a bit confusing if the way it's written is intended because it's `if ignore this error then throw the error`

Reply:
@jules2689 Great point, I just refactored this PR in general because the check was in the wrong place but I still tried to clean up the wording. The latest version filters through all of the errors and checks if any of them has a "non-ignored" error type, and then throws the error. 

https://github.com/integrations/jira/blob/a56843186404eb1f552d155a0643fe89b0dd5b28/lib/sync/installation.js#L105-L112